cli/smartcontract/smart_contract.go

```diff
@@ -337,6 +337,21 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:   "calc-hash",
+				Usage:  "calculates hash of a contract after deployment",
+				Action: calcHash,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "sender, s",
+						Usage: "sender script hash or address",
```

Comment:
Maybe add here that script hash should be in LE?

Reply:
All user-facing things are usually LE.